packages/core/contracts/external/ovm/OVM_CrossDomainEnabled.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is this a new contract or was it moved from somewhere?

Reply:
Ah copied from [here](https://github.com/ethereum-optimism/optimism/blob/5fc728da7381eda6b6f858c091636c947186a3ce/packages/contracts/contracts/optimistic-ethereum/libraries/bridge/OVM_CrossDomainEnabled.sol). I added a comment.